test/e2e/kam_test.go

```diff
@@ -38,12 +40,13 @@ func validateKamService(t *testing.T) {
 
 	// check ConsoleLink
```

Comment:
The comment is now incorrect, it should say something like // check CLI download link